share/revng/test/tests/abi/gather_symbols.py

```diff
@@ -99,7 +99,59 @@ def find_callsites(self, name):
 
 class DumpbinOutputParser:
```

Comment:
If there's a common base API between the two parser, create a base class inheriting from `ABC` and use `@abstractmethod`: https://docs.python.org/3/library/abc.html#abc.abstractmethod

Reply:
Done.